weave

```diff
@@ -261,14 +261,14 @@ elif [ "$1" = "run" -a -z "$IS_LOCAL" ] ; then
     # we can't rely on our baked-in docker to support those arguments.
     shift 1
     if [ "$1" != "--without-dns" ] ; then
-        DNS_ARGS=$(exec_remote dns-args "$@" || true)
+        eval "$(exec_remote dns-args "$@" 2>/dev/null || true)"
```

Comment:
That's weird since it is exactly what `with_container_fqdn` does, which too is called in attach.